x-pack/plugins/data_enhanced/server/search/eql_search_strategy.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger } from 'kibana/server';
+import { ApiResponse, TransportRequestPromise } from '@elastic/elasticsearch/lib/Transport';
+
+import {
+  getAsyncOptions,
+  getDefaultSearchParams,
+  ISearchStrategy,
+  toSnakeCase,
+  shimAbortSignal,
+} from '../../../../../src/plugins/data/server';
+import { EqlSearchStrategyRequest, EqlSearchStrategyResponse } from '../../common/search/types';
+
+export const eqlSearchStrategyProvider = (
+  logger: Logger
+): ISearchStrategy<EqlSearchStrategyRequest, EqlSearchStrategyResponse> => {
```

Comment:
@rylnd I still suspect the two strategies are identical, except for the endpoint being used.
Don't you think we could do with adding a parameter to the normal strategy and using that one?

Lets talk about this.